imports/ui/pages/player/player.js

```diff
@@ -96,6 +96,15 @@ Template.player.onCreated(function () {
   // Description
   this.playerState.set('showDescription', false)
 
+  this.togglePlay = () => {
+    const dict = this.playerState
```

Comment:
ok for now, but note that we will refactor the player to have more sane and standardized state- and event management